technical-doc/api.rst

```diff
@@ -23,13 +23,15 @@ Massa JSON-RPC API is splitted in two parts :
 Find the complete Massa `OpenRPC <https://spec.open-rpc.org/>`_ specification `here
 <https://raw.githubusercontent.com/massalabs/massa/main/massa-node/base_config/openrpc.json>`_.
 
+.. note::
```

Comment:
I'd put a bit more more context. Maybe something like this: `We provide an <link: Interactive API specification> which documents every endpoints available and allows you to play with the API`.

Reply:
Yep